vendor/json-stringify-safe/stringify.js

```diff
@@ -0,0 +1,27 @@
+exports = module.exports = stringify
```

Comment:
Can we use this as an actual dependency in `package.json` now? I don't even know how that works and we don't have any dependencies yet, so I opted to just vendor. Ideally, we'd just point to this package on npm.